### DISCUSSION TITLE
[Bug]: Telemetry Data only visible in iOS app for Bluetooth connected device

### DISCUSSION THREAD

**Discussion by spqr1234** [meshtastic/firmware#4074] [Troubleshooting]
### Category

Other

### Hardware

Rak4631

### Firmware Version

2.3.11

### Description

Having installed the BOSCH BME680 environment sensor and configured the iOS app to enable the Telemetry sensor module.  The telemetry data only appears in the node info if directly connected to the device via bluetooth.  When I connect using the iOS app to other nodes on the same mesh the telemetry data of the node on which the module is installed is not being displayed in the node info.  It seems the telemetry data is not being shared via LoRa

**Comment by thebentern** [comment]
Telemetry is only broadcasted on the primary (first) channel. Are the other nodes on this channel?

**Reply by spqr1234** [reply]
I have shared the same channels between all devices. Channel 0 = 'LongFast' Key = 'AQ=='

**Reply by spqr1234** [reply]
I see now what the issue was.  When connected via Bluetooth the telemetry is updated every few minutes or so, whereas it's limited by the update interval in the Telemetry settings in the App over LoRa

**Reply by thebentern** [reply]
Ah yes, that is correct. We can spare the faster updates over bluetooth vs LoRA traffic, so we send those more frequently to the phone. :-)
